src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -721,15 +716,7 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                 this,
                 diagnostics);
 
-            var ctor = (MethodSymbol)GetWellKnownTypeMember(this.Compilation, TupleTypeSymbol.GetTupleCtor(elements.Length), diagnostics, syntax: node);
-            if ((object)ctor != null)
-            {
-                ctor = ctor.AsMember(type.UnderlyingTupleType);
-            }
-            else
-            {
-                hasErrors = true;
-            }
+            var ctor = type.GetTupleConstructorChain(node, this, diagnostics, ref hasErrors);
```

Comment:
Please spell-out the type. In general, we disallow use of ```var``` unless the type is mentioned in the initializer. For example, "var builder = **ArrayBuilder\<int\>**.GetInstance();" is Ok, but this statement is not Ok. Usage of var in this manner negatively affects readability of the code.